meilisearch-lib/src/dump/compat/v3.rs

```diff
@@ -186,10 +186,10 @@ impl Failed {
 impl From<(UpdateStatus, String, TaskId)> for Task {
     fn from((update, uid, task_id): (UpdateStatus, String, TaskId)) -> Self {
         // Dummy task
-        let mut task = Task {
+        let mut task = super::v4::Task {
             id: task_id,
-            index_uid: IndexUid::new(uid).unwrap(),
-            content: TaskContent::IndexDeletion,
+            index_uid: IndexUid::new_unchecked(uid),
```

Comment:
Why not unwrapping here? it is incorrect to not check that.

Reply:
Ok I understand, we shouldn't be more restrictive than before, but I understand the point. Thank you for the explanation.